content/en/api/integrations_webhook/code_snippets/custom_variables_update.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+api_key="<DATADOG_API_KEY>"
+app_key="<DATADOG_APPLICATION_KEY>"
+
+curl -v -X PUT \
+-H "DD-API-KEY: ${api_key}" \
+-H "DD-APPLICATION-KEY: ${app_key}" \
+"https://api.datadoghq.com/api/v1/webhooks/configuration/webhooks/<CUSTOM_VARIABLE_NAME>" \
+-d '{
+    "name": "<NEW_CUSTOM_VARIABLE_NAME>",
+    "is_secret": true,
+}'
```

Comment:
Isn't the value that can be updated?

Reply:
Ahh i guess both, let's update the value in the example. `name` is a bit of a special case since it's also the resource identifier.